internal/backend/runtime/omni/controllers/omni/machine_status.go

```diff
@@ -249,7 +249,7 @@ func (ctrl *MachineStatusController) reconcileRunning(ctx context.Context, r con
 
 		spec.Connected = connected
 
-		helpers.CopyLabels(machine, m, omni.LabelMachineRequest, omni.LabelMachineRequestSet, omni.LabelNoManualAllocation)
+		helpers.CopyLabels(machine, m, omni.LabelMachineRequest, omni.LabelMachineRequestSet, omni.LabelNoManualAllocation, omni.LabelIsManagedByStaticInfraProvider)
```

Comment:
Propagate this label  from Machine to MachineStatus. We then check this label on `MachineStatus` in `ClusterMachineConfigStatusController` to find out if the machine is being prepared by the static infra provider (still not ready to be used, e.g., being wiped), and if it is, skip applying config to it.